draft-ietf-webpush-protocol.xml

```diff
@@ -772,8 +774,8 @@ HEADERS      [stream 7] +END_STREAM +END_HEADERS
       <figure>
         <preamble>
           The push service permits the request to remain outstanding.  When a
-          push message is sent by an application server, a server push is
-          associated with the initial request.  The response includes the push
+          push message is sent by an application server, a server push is generated
+          in association with the initial request.  The server push's response includes the push

```

Comment:
maybe instead "the response for the server push includes".  Possessiveness is something we try to avoid using for abstract concepts.